learn/dmn.html.haml

```diff
@@ -92,3 +92,13 @@ layout: learnTopicBase
       We contribute to the
       %a{ :href => "https://dmn-tck.github.io/tck", :title => "DMN TCK" } DMN Technology Compatibility Kit (DMN TCK)
       %span> , a community-led proposal for a verifiable and executable method to demonstrate the Conformance level of support provided by a Vendor supplied DMN implementation.
+
+%h2 Resources
+%p
+  Looking for a gentle introduction to the DMN standard?
+  %br
+  We have just the right crash course on DMN, freely available for you at:
+  %a{ :href => "http://learn-dmn-in-15-minutes.com", :title => "Learn DMN in 15 minutes .com" }Learn DMN in 15 minutes .com
```

Comment:
Generally, I prefer to use the **learn-dmn-in-15-minutes.com** form.
```suggestion
  %a{ :href => "http://learn-dmn-in-15-minutes.com", :title => "learn-dmn-in-15-minutes.com" }learn-dmn-in-15-minutes.com
```
But, feel free to keep it as is ;-)